### PR TITLE
Frontend-ci-yml-add

### PR DESCRIPTION
**Pull request by avantitandon**
Trying to add this!

### CODE REVIEW

**Review by BardyBard — APPROVED** [review]
Looks good Tara!

File: .github/workflows/Ci.yml
Comment (BardyBard) [inline .github/workflows/Ci.yml:1]: Could we give this a better file name plz

Comment (avantitandon) [reply .github/workflows/Ci.yml:1]: no <3

Comment (BardyBard) [reply .github/workflows/Ci.yml:1]: its a serious comment 😭 can you please rename it to something more descriptive, like frontend-tests.yml or whatever you think is good

**Review by solarescarlos — APPROVED** [review]
Thanks for adding the tests Tara.

**Pull request merged**